specification/compliance.md

```diff
@@ -0,0 +1,19 @@
+# FOCUS Compliance
+
+Most billing datasets are unique based on the provider, its method of determining cost (e.g. fixed vs consumption-based billing), support for various concepts (e.g. commitment-based discounts), and the provider's billing system's maturity level.
+
+FOCUS aims to help FinOps practitioners perform common FinOps capabilities by using a generic set of instructions. FOCUS accomplishes this by defining compliance requirements that account for the provider-level differences mentioned above.
+
+## Enabling FinOps Capabilities For Heterogeneous Cost Datasets
+
+Cloud Service Providers (CSPs) generate detailed cost and usage data in near real-time enabling an extensive set of FinOps capabilities to drive accountability and efficiency for dynamic consumption-based costs.  Non-CSP providers typically provide billing datasets that lack the same level of detail. The purpose of FOCUS Essential's reduced data set is to allow easier access for non-CSP providers to be incorporated in the full FOCUS data cube without breaking any primary FinOps capabilities, performing invoice reconciliation, data ingestion, and data normalization. Simplified versions of additional FinOps capabilities like cost reporting, chargeback, showback, and budgeting/forecasting can also be performed using a less-detailed billing dataset when essential requirements are met.
```

Comment:
```suggestion
Cloud Service Providers (CSPs) and some SaaS providers generate detailed cost and usage data in near real-time enabling an extensive set of FinOps capabilities to drive accountability and efficiency for dynamic consumption-based costs.  Non-CSP providers typically provide billing datasets that lack the same level of detail. The purpose of FOCUS Essential's reduced data set is to allow easier access for non-CSP providers to be incorporated in the full FOCUS data cube without breaking any primary FinOps capabilities, performing invoice reconciliation, data ingestion, and data normalization. Simplified versions of additional FinOps capabilities like cost reporting, chargeback, showback, and budgeting/forecasting can also be performed using a less-detailed billing dataset when essential requirements are met.
```

Reply:
Reworded the whole thing.. captured your point about some SaaS.